src/main/webapp/app/meta/meta.component.html

```diff
@@ -1,61 +1,70 @@
-                        </span>
-                    </td>
-                </tr>
-            </tbody>
-        </table>
+    <div [hidden]="loadingIndicator">
+        <div>
+            <input type="text" class="filter-input" placeholder="Search..." (keyup)='updateFilter($event)'/>
+            <button class="btn btn-primary button-margin download-button" (click)="download()">Download</button>
+        </div>
+        <ngx-datatable #table class="bootstrap"
+                       [rows]="rows"
+                       [columnMode]="'flex'"
+                       [headerHeight]="50"
+                       [footerHeight]="50"
+                       [rowHeight]="'auto'"
+                       [limit]="15">
+            <ngx-datatable-column prop="protocol_no" name="Protocol No" [flexGrow]="1">
+                <ng-template let-value="value" ngx-datatable-cell-template>
+                    <span class="button-margin">{{ value }}</span>
+                    <a target="_blank" href="https://www.mskcc.org/cancer-care/clinical-trials/{{value}}"><i class="fa fa-external-link" aria-hidden="true"></i></a>
+                </ng-template>
+            </ngx-datatable-column>
+            <ngx-datatable-column prop="nct_id" name="Nct Id" [flexGrow]="1">
+                <ng-template let-value="value" ngx-datatable-cell-template>
+                 <span *ngIf="value.length > 0">
+                    <span class="button-margin">{{ value }}</span>
+                    <a target="_blank" href="https://clinicaltrials.gov/ct2/show/{{value}}"><i class="fa fa-external-link" aria-hidden="true"></i></a>
+                </span>
+                </ng-template>
+            </ngx-datatable-column>
+            <ngx-datatable-column name="Status" [flexGrow]="1">
+                <ng-template let-value="value" ngx-datatable-cell-template>
+                    {{value}}
+                </ng-template>
+            </ngx-datatable-column>
+            <ngx-datatable-column name="Title" [flexGrow]="3">
+                <ng-template let-value="value" ngx-datatable-cell-template>
+                    {{value}}
+                </ng-template>
+            </ngx-datatable-column>
+            <ngx-datatable-column prop="precision_medicine" name="Precision Medicine" [flexGrow]="1">
+                <ng-template ngx-datatable-cell-template let-rowIndex="rowIndex" let-row="row" let-value="value">
+                    <form>
+                        <input type="radio" value="YES" name="precision-medicine" class="radio-margin" [(ngModel)]="rows[rowIndex]['precision_medicine']" (click)="updatePrecisionMedicine('precision_medicine', $event, row, rowIndex)">YES
+                        <input type="radio" value="NO" name="precision-medicine" class="radio-margin" [(ngModel)]="rows[rowIndex]['precision_medicine']" (click)="updatePrecisionMedicine('precision_medicine', $event, row, rowIndex)">NO
+                    </form>
+                    <!--<span-->
```

Comment:
We don't need these anymore?

Reply:
Sorry, I forgot to remove commented content.